### ISSUE TITLE
Disable the AutoComplete feature (input, textarea)

### ISSUE THREAD

**Issue by MK-RD** [AdvancedCustomFields/acf#531]
For content management (admin area) it would make sense to prevent the web browser (MF) from remembering the input value on refresh. This can be done, for example, by deactivating the AutoComplete feature for input and textarea elements.

**autocomplete="off"**

If this is already known, I would be very interested in why this was not implemented?

Thanks in advance!

**Comment by MK-RD** [comment]
Hi @lgladdy 

Always these difficult questions :-)

Even if this (e.g. [title field](https://core.trac.wordpress.org/browser/tags/5.8/src/wp-admin/edit-form-advanced.php#L544)) is implemented by default in WordPress without a filter option, I cannot rule out that a filter at ACF field level could be useful for someone.

At this point one could also consider enabling the filtering of all attributes of an HTML element (input, textarea ...) of a specific ACF field.

In summary, I would recommend this as a default setting with the option of manipulation using a filter.

Regards
Matt

**Comment by lgladdy** [comment]
Cheers @MK-RD. I'll discuss with the team here on if they think a filter is necessary, and we'll get this added to the feature requests list.

**Comment by MK-RD** [comment]
Hi @lgladdy

Is there any new information?

Regards
Matt
